src/App.tsx

```diff
@@ -1,109 +1,33 @@
 import React from 'react';
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
+import { User } from './types/User';
+import { Comment } from './types/Comment';
+import { Post } from './types/Post';
+import { PostList } from './components/PostList';
 
 import './App.scss';
 
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+function getUser(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
```

Comment:
you can do it in this way (as you do in `getComment`):
 ```suggestion
  return usersFromServer.find(user => user.id === userId) || null;
```